diff -Naur verilog-arbiter/src/arbiter.v verilog-arbiter.tcm/src/arbiter.v
--- verilog-arbiter/src/arbiter.v	2020-05-17 11:04:06.222824414 -0500
+++ verilog-arbiter.tcm/src/arbiter.v	2020-05-17 11:06:10.331164104 -0500
@@ -54,7 +54,9 @@
             ff1 = 0;
             for (i = NUM_PORTS-1; i >= 0; i=i-1) begin
                 if (in[i])
+		/* verilator lint_off WIDTH */ 
                     ff1 = i;
+		/* verilator lint_off WIDTH */ 
             end
         end
     endfunction
